.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
I'm not sure about this description. "Create a report of a bug" would fit. I'm not sure about the improving aspect of it.